implementations/elixir/ockam/ockam_services/lib/services/authorization_config.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Ockam.Services.AuthorizationConfig do
```

Comment:
This can possibly the beginning of a policies layer and maybe this module could be called `AuthorizationPolicy`